tutorial.md

```diff
@@ -0,0 +1,305 @@
+<h1 align="center">
+  Astro Python SDK
+</h1>
+  <h3 align="center">
+  Tutorial<br><br>
+</h3>
+
+# Introduction
+
+This tutorial demonstrates how to use the Astro Python SDK through a simple ETL example that you can run on your local machine:
+* **Extract** a file from S3 into a Snowflake relational table
+* **Transform** that table in Snowflake
+* **Load** that transformed table into a reporting table in Snowflake
+
+You'll need to use existing Amazon S3 and Snowflake accounts, or create some trial versions.
+
+***
+# Let's do some setup
+
+## Install Airflow
+
+For macOS, you can follow these steps:
+* Install homebrew: 
+
+    `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+* `brew install pip`
+* `brew install python`
+* Install Airflow:
+
+    `pip3 install apache-airflow --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.2.5/constraints-3.9.txt"`
+* Initialize Airflow: 
+
+    `airflow db init`
+
+## Set up your data stores
+
+### Set up S3
+
+(TODO: add some generic instructions for setting up a trial S3 instance)
+
+Upload this datafile from your local machine to your newly created aws bucket
+
+```
+order_id,customer_id,purchase_date,amount
+ORDER1,CUST1,1/1/2021,100
+ORDER2,CUST2,2/2/2022,200
+ORDER3,CUST3,3/3/2023,300
+```
+
+### Set up Snowflake
+
+(TODO: Add some generic instructions for setting up a trial Snowflake instance)
+
+## Install Astro on your local machine
+
+* [Install the Astro CLI](https://docs.astronomer.io/astro/install-cli), following the additional steps to [create an Astro project.](https://docs.astronomer.io/astro/create-project)
+
+* In the project directory you just created, create a `.env` file using your favorite file editor:
+
+```shell
+AIRFLOW__CORE__ENABLE_XCOM_PICKLING=True
+AIRFLOW__ASTRO__SQL_SCHEMA=<snowflake_schema>
+```
+
+
+* Note: prior to running `astrocloud dev start` you'll need to navigate to your Docker Desktop application and double-click on it.
```

Comment:
Do we need this step?
How does the local Python environment (in which we installed dependencies) relate to this Docker container?

Reply:
If I try to run `astrocloud dev start` without starting up docker, I get this error:
![Screen Shot 2022-05-20 at 11 27 18 AM](https://user-images.githubusercontent.com/4237498/169591781-19269d10-050c-447b-8305-7f3994a8fa96.png)

Reply:
Do we need `astrocloud dev start`?
Would it be worth using `airflow standalone` and reducing the dependencies for this first tutorial?